src/encoded/static/components/file.js

```diff
@@ -324,6 +324,13 @@ class FileComponent extends React.Component {
                                     <dd>{context.output_type}</dd>
                                 </div>
 
+                                {context.index_of ?
+                                <div data-test="outputtype">
+                                    <dt>Index of</dt>
+                                    <dd>{context.index_of.map(file => file.replace(/\/files\/|\//g, '')).join(', ')}</dd>
+                                </div>
+                                : null }
+
```

Comment:
A style comment: in order to match the other blocks of code here, I would indent lines 328 - 331 and I would get rid of the space after "null" in line 332.

Reply:
Done. Thanks.